src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -215,12 +219,30 @@ const AccountTransfer = (() => {
                         return;
                     }
                     withdrawal_limit = +response_limits.get_limits.remainder;
+
                     if (withdrawal_limit < +min_amount) {
                         getElementById(messages.limit).setVisibility(1);
                         getElementById(messages.parent).setVisibility(1);
                         return;
                     }
-                    getElementById('range_hint').textContent = `${localize('Min')}: ${min_amount} ${localize('Max')}: ${client_balance <= withdrawal_limit ? localize('Current balance') : localize('Withdrawal limit')}`;
+                    max_amount = Currency.getMaxTransfer(
+                        Client.get('currency')
+                    );
+                    transferable_amount = max_amount ?
+                        Math.min(max_amount, withdrawal_limit, client_balance) :
+                        Math.min(withdrawal_limit, client_balance);
+
+                    getElementById('range_hint').textContent = `${localize('Min')}: ${min_amount} ${localize('Max')}: ${transferable_amount.toFixed(Currency.getDecimalPlaces(Client.get('currency')))}`;
```

Comment:
same here, can min and max be set in the template and we just add their values via javascript?

Reply:
Ah, I see what you mean, so something like: 
```
<div id=range_hint>
   {it.L('Min:'}<span id='account_transfer_min_value'></span>
   {it.L('Max:'}<span id='account_transfer_max_value'></span>
</div>
```
?

Reply:
that's right